<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PythonEditor</class>
 <widget class="QMainWindow" name="PythonEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>734</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Cloud Compare - Python Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../CloudCompare-PythonRuntime.qrc">
    <normaloff>:/CC/plugin/PythonRuntime/images/python-editor-icon.png</normaloff>:/CC/plugin/PythonRuntime/images/python-editor-icon.png</iconset>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenFolder"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>&amp;Run</string>
    </property>
    <addaction name="actionRun"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionComment"/>
    <addaction name="actionUncomment"/>
    <addaction name="actionIndentMore"/>
    <addaction name="actionIndentLess"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="separator"/>
   <addaction name="actionComment"/>
   <addaction name="actionUncomment"/>
   <addaction name="separator"/>
   <addaction name="actionIndentMore"/>
   <addaction name="actionIndentLess"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="projectBrowser">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>146</width>
     <height>319</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Project Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout">
     <item>
      <widget class="ProjectView" name="projectTreeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>126</width>
         <height>234</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="scriptOutputConsoleDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Script Output Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents2">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QListWidget" name="scriptOutputConsole"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="viewsToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../CloudCompare-PythonRuntime.qrc">
     <normaloff>:/CC/plugin/PythonRuntime/images/editor/plus.png</normaloff>:/CC/plugin/PythonRuntime/images/editor/plus.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../CloudCompare-PythonRuntime.qrc">
     <normaloff>:/CC/plugin/PythonRuntime/images/editor/ccOpen.png</normaloff>:/CC/plugin/PythonRuntime/images/editor/ccOpen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../CloudCompare-PythonRuntime.qrc">
     <normaloff>:/CC/plugin/PythonRuntime/images/editor/ccSave.png</normaloff>:/CC/plugin/PythonRuntime/images/editor/ccSave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close and Return to Cloud Compare</string>
   </property>
   <property name="toolTip">
    <string>Close and Return to Cloud Compare</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="../CloudCompare-PythonRuntime.qrc">
     <normaloff>:/CC/plugin/PythonRuntime/images/editor/arrow_right.png</normaloff>:/CC/plugin/PythonRuntime/images/editor/arrow_right.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../CloudCompare-PythonRuntime.qrc">
     <normaloff>:/CC/plugin/PythonRuntime/images/editor/ccSegment.png</normaloff>:/CC/plugin/PythonRuntime/images/editor/ccSegment.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../CloudCompare-PythonRuntime.qrc">
     <normaloff>:/CC/plugin/PythonRuntime/images/editor/copy.png</normaloff>:/CC/plugin/PythonRuntime/images/editor/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../CloudCompare-PythonRuntime.qrc">
     <normaloff>:/CC/plugin/PythonRuntime/images/editor/paste.png</normaloff>:/CC/plugin/PythonRuntime/images/editor/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>&amp;Select All</string>
   </property>
  </action>
  <action name="actionReset_Chai_to_initial_state">
   <property name="text">
    <string>Reset Chai to initial state</string>
   </property>
  </action>
  <action name="actionSave_Chai_state">
   <property name="text">
    <string>Save Chai state</string>
   </property>
  </action>
  <action name="actionReset_chai_to_last_save">
   <property name="text">
    <string>Reset chai to last save</string>
   </property>
  </action>
  <action name="actionComment">
   <property name="icon">
    <iconset resource="../CloudCompare-PythonRuntime.qrc">
     <normaloff>:/CC/plugin/PythonRuntime/images/editor/comment.png</normaloff>:/CC/plugin/PythonRuntime/images/editor/comment.png</iconset>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K, Ctrl+C</string>
   </property>
  </action>
  <action name="actionUncomment">
   <property name="icon">
    <iconset resource="../CloudCompare-PythonRuntime.qrc">
     <normaloff>:/CC/plugin/PythonRuntime/images/editor/uncomment.png</normaloff>:/CC/plugin/PythonRuntime/images/editor/uncomment.png</iconset>
   </property>
   <property name="text">
    <string>Uncomment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K, Ctrl+U</string>
   </property>
  </action>
  <action name="actionIndentMore">
   <property name="icon">
    <iconset resource="../CloudCompare-PythonRuntime.qrc">
     <normaloff>:/CC/plugin/PythonRuntime/images/editor/indent_more.png</normaloff>:/CC/plugin/PythonRuntime/images/editor/indent_more.png</iconset>
   </property>
   <property name="text">
    <string>Indent More</string>
   </property>
  </action>
  <action name="actionIndentLess">
   <property name="icon">
    <iconset resource="../CloudCompare-PythonRuntime.qrc">
     <normaloff>:/CC/plugin/PythonRuntime/images/editor/indent_less.png</normaloff>:/CC/plugin/PythonRuntime/images/editor/indent_less.png</iconset>
   </property>
   <property name="text">
    <string>Indent Less</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="icon">
    <iconset resource="../CloudCompare-PythonRuntime.qrc">
     <normaloff>:/CC/plugin/PythonRuntime/images/editor/ccOpen.png</normaloff>:/CC/plugin/PythonRuntime/images/editor/ccOpen.png</iconset>
   </property>
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProjectView</class>
   <extends>QTreeView</extends>
   <header>ProjectView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../CloudCompare-PythonRuntime.qrc"/>
 </resources>
 <connections/>
</ui>
